prowler/lib/outputs/models.py

```diff
@@ -228,6 +229,18 @@ def unroll_dict(dict: dict):
     return unrolled_items
 
 
+def unroll_dict_to_list(dict: dict):
```

Comment:
Could you please include tests for this function? And move it to another file to be able to use from another places.